src/blockchain_db/sqlite/db_sqlite.cpp

```diff
@@ -791,17 +794,20 @@ bool BlockchainSQLite::return_staked_amount_to_user(
         int64_t payout_height = height + (delay_blocks > 0 ? delay_blocks : 1);
         auto insert_payment = prepared_st(
                 "INSERT INTO delayed_payments (eth_address, amount, payout_height, entry_height) "
-                "VALUES (?, ?, ?, ?)");
+                "VALUES (?, ?, ?, ?) "
+                "ON CONFLICT(eth_address, payout_height) DO UPDATE SET amount = (amount + excluded.amount);");
```

Comment:
for my own knowledge, where does the `excluded` key come from?

Reply:
It's a reference to the conflicting data in an `ON CONFLICT` clause that couldn't be inserted.  Unprefixed columns refer to the value currently in the table (i.e. the one preventing the insertion), and `excluded.` columns refer to the new data.  See https://www.sqlite.org/lang_upsert.html

Reply:
I definitely missed this checking their docs, thank ya